### DISCUSSION TITLE
这个项目为什么要用 leveldb作为默认存储

### DISCUSSION THREAD

**Discussion by wmvm0** [Mrs4s/go-cqhttp#1878] [General]
这个项目为什么要用 leveldb作为默认存储，有什么优势吗

**Comment by Ink-33** [comment]
早期只有leveldb好使，go没有纯go的sqlite3驱动，只有用ccgo从c生成的go代码，兼容性有点问题。现在gocq已经支持sqlite了

**Reply by wmvm0** [reply]
了解
